nautobot/utilities/templatetags/helpers.py

```diff
@@ -317,19 +317,7 @@ def get_docs(model):
         >>> get_docs(obj)
         "some text"
     """
-    path = "{}/models/{}/{}.md".format(settings.DOCS_ROOT, model._meta.app_label, model._meta.model_name)
-    try:
-        with open(path, encoding="utf-8") as docfile:
-            content = docfile.read()
-    except FileNotFoundError:
-        return "Unable to load documentation, file not found: {}".format(path)
-    except IOError:
-        return "Unable to load documentation, error reading file: {}".format(path)
-
-    # Render Markdown with the admonition extension
-    content = markdown(content, extensions=["admonition", "fenced_code", "tables"])
-
-    return mark_safe(content)
+    return f"{settings.STATIC_URL}{f'{model._meta.app_label}/' if model._meta.app_label in settings.PLUGINS else ''}docs/models/{model._meta.app_label}/{model._meta.model_name}.html"
```

Comment:
Hm. Is there a technical reason why we need to have plugin docs in `app_label/docs/models/app_label/` while core docs can just be in `docs/models/app_label/`? Plugin `app_label` should always be different from any core `app_label` in any case.

Reply:
Yeah, so maybe this is just by convention, but anything in static files in plugins are stored in `static/app_label`. I could just make the plugin docs `app_label/docs/models/model_name`, but then it doesn't follow the same convention as core, and it would make for a longer line.  But I am open to suggestions.